### PR TITLE
Combine consecutive issets

### PR DESCRIPTION
**Pull request by MathiasReker**
Using isset($var) && multiple times should be done in one call.

### CODE REVIEW

File: manual/vote-note.php [hunk 1]
```diff
@@ -42,7 +42,7 @@
         if (json_last_error() == JSON_ERROR_NONE && isset($r->status) && $r->status && isset($r->votes)) {
           $response["success"] = true;
           $response["update"] = (int)$r->votes;
-        } elseif (json_last_error() == JSON_ERROR_NONE && isset($r->status) && isset($r->message) && $r->status == false) {
```
Comment (kamil-tekiela) [inline manual/vote-note.php:45]: Please trim whitespace in both places.

File: my.php [hunk 2]
```diff
@@ -10,7 +10,7 @@
 $options = array();
 
 // We have post data, and it is an available language
-if (isset($_POST['my_lang']) && isset($langs[$_POST['my_lang']])) {
+if (isset($_POST['my_lang'], $langs[$_POST['my_lang']])  ) {
```
Comment (localheinz) [inline my.php:13]:
```suggestion
if (isset($_POST['my_lang'], $langs[$_POST['my_lang']])) {
```

File: manual/vote-note.php [hunk 3]
```diff
@@ -42,7 +42,7 @@
...
+        } elseif (json_last_error() == JSON_ERROR_NONE && isset($r->status, $r->message)   && $r->status == false) {
```
Comment (localheinz) [inline manual/vote-note.php:45]:
```suggestion
        } elseif (json_last_error() == JSON_ERROR_NONE && isset($r->status, $r->message) && $r->status == false) {
```

File: manual/vote-note.php (hunk 1 above)
Comment (MathiasReker) [reply manual/vote-note.php:45]: Thanks @localheinz

**Comment by cmb69** [comment]
Thank you!

**Comment by localheinz** [comment]
For reference, see https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v3.8.0/doc/rules/language_construct/combine_consecutive_issets.rst.

**Pull request merged**